pkg/cmd/cli/cmd/status.go

```diff
@@ -59,11 +60,15 @@ type StatusOptions struct {
 // NewCmdStatus implements the OpenShift cli status command.
 func NewCmdStatus(name, fullName string, f *clientcmd.Factory, out io.Writer) *cobra.Command {
 	opts := &StatusOptions{}
+	cmdName := fullName
+	if strings.HasSuffix(fullName, name) {
```

Comment:
@liggitt Without this, the help output for this command would include `status` in cases where it shouldn't:

`openshift cli status -h` 
```
...
This command will show services, deployment configs, build configurations, and active deployments.
If you have any misconfigured components information about them will be shown. For more information
about individual items, use the describe command (e.g. openshift cli status describe buildConfig,
openshift cli status describe deploymentConfig, openshift cli status describe service).
```